YourFirstPR.md

```diff
@@ -75,7 +75,7 @@ bundle show fastlane
 
 which should print out the path to your local development environment.
 
-From now on, every time you introduce a change to your local _fastlane_ code base, you can immediately test it by running `bundle exec fastlane …`.
+From now on, every time you introduce a change to your local _fastlane_ code base, you can immediately test it by running  (`bundle exec fastlane …`. (Note that just using `fastlane …` without `bundle exec` will **not** use your local _fastlane_ code base!)
```

Comment:
I think you snuck an extra `( ` in there at the beginning 😅

Reply:
Good catch 🎣 Thanks.